plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -419,7 +419,7 @@ public String toString() {
 
     public static class DiskDef {
         public enum DeviceType {
-            FLOPPY("floppy"), DISK("disk"), CDROM("cdrom");
+            FLOPPY("floppy"), DISK("disk"), CDROM("cdrom"), LUN("lun");
```

Comment:
SCSI block devices, LUN are not supported; the type was added to avoid possible bug in case we decide to support this storage/device type in future. See https://libvirt.org/formatdomain.html#elementsDisks for details.